org.xaraxtreme.XaraLX.json

```diff
@@ -0,0 +1,779 @@
+{
+    "app-id": "org.xaraxtreme.XaraLX",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "xaralx",
+    "rename-icon": "xaralx",
+    "rename-desktop-file": "xaralx.desktop",
+    "rename-appdata-file": "xaralx.appdata.xml",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
```

Comment:
Graphics editor with sound support?

Reply:
It can be built with a built-in movie player, based on [wxMediaCtrl](http://docs.wxwidgets.org/2.8/wx_wxmediactrl.html#wxmediactrl) control. The original purpose for this was to display [video tutorials](http://www.xaraxtreme.org/about-movies.html). However, it is disabled in this version and the flatpak package does not provide any video files.